hacs.json

```diff
@@ -1,9 +1,9 @@
 {
-    "name": "Solarfocus eco manager-touch",
+    "name": "Solarfocus eco manager-touch for eco top light",
```

Comment:
According to how I understand Solarfocus' documentation, the eco manager-touch is included in different products [link](https://www.solarfocus.com/de/partnerportal/pdf/open/UGFydG5lcmJlcmVpY2gtREUvUmVnZWx1bmdfZWNvbWFuYWdlci10b3VjaC9BbmxlaXR1bmdlbi9lY29tYW5hZ2VyLXRvdWNoX01vZGJ1cy1UQ1AtUmVnaXN0ZXJkYXRlbl9BbmxlaXR1bmcucGRm/117920/0/Lng_YSxpM245S30zMTc4W2Y8cVRRXWlJVWRQJDsv?serialNumber=68332 )

Reply:
Yes, the link is good - can we place it somewhere in the repository documentation?

Reply:
its on the README under software versions

Reply:
```suggestion
    "name": "Solarfocus eco manager-touch",
```